src/gpu/generic/sycl/ref_softmax.cpp

```diff
@@ -36,11 +36,29 @@ status_t ref_sycl_softmax_fwd_t::pd_t::init_conf() {
     conf_.outer_size = outer_size();
     conf_.channels = axis_size();
     conf_.wk_size = inner_size() * outer_size();
+
+    for (int i = 0; i < DNNL_MAX_NDIMS; i++) {
+        conf_.dst_dims[i] = dst_md()->dims[i];
+    }
+    conf_.dst_ndims = dst_md()->ndims;
```

Comment:
That info is alreadin in conf as a part of dst_md. Is there any good reason to duplicating it?

Reply:
Thanks for the suggestion.

Reply:
I menant this for `conf_.dst_dims` as well, not just `conf_.dst_ndims`.

Reply:
Sorry. I just missed that one. I did the changes now. Thanks.